assets/common/components/mlflow_model_local_validation/spec.yaml

```diff
@@ -17,6 +17,7 @@ command: >-
   --model-path ${{inputs.model_path}}
   $[[--test-data-path ${{inputs.test_data_path}}]]
   $[[--column-rename-map "${{inputs.column_rename_map}}"]]
+  $[[--task-name ${{inputs.task_name}}]]
```

Comment:
could we read task_type from MLmodel? both OSS and hftrasnsformers v2 has task emitted to MLModel to be able to load in future

Reply:
@novaturient95 What about models that support multiple tasks?

Reply:
@skasturi for that scenario as well, we can or should infer from MLmodel especially if its getting logged in case of OSS transformers as well as hftv2